protocol/avdl/stellar1/local.avdl

```diff
@@ -35,6 +35,7 @@ protocol local {
   array<CurrencyLocal> getDisplayCurrenciesLocal(int sessionID);
 
   void changeWalletAccountNameLocal(int sessionID, AccountID accountID, string newName);
+  void deleteWalletAccountLocal(int sessionID, AccountID accountID);
```

Comment:
This is a particularly sensitive operation so if you would like to add the following safe guard could be nice. It would make it less likely for someone to accidentally build an interface that does not prompt before deletion. I leave it up to you.

Add a parameter `userConfirmation string` and in the RPC assert that it is at least the first 5 digits (case insensitive) of the account ID. The CLI and GUI will probably ask for something like that as part of their process. The specifics of the confirmation string can easily be changed to match the gui.

Reply:
good idea